starport/cmd/module-create.go

```diff
@@ -111,6 +125,30 @@ func createModuleHandler(cmd *cobra.Command, args []string) error {
 		fmt.Println(sourceModificationToString(sm))
 	}
 
+	if len(dependencies) > 0 {
+		dependencyWarning(dependencies)
```

Comment:
It's possible to move this `dependencyWarning` method call to line 101?

```golang
	if len(dep) > 0 {
		dependencies = strings.Split(dep, ",")
		options = append(options, scaffolder.WithDependencies(dependencies))
		dependencyWarning(dependencies)
	}
```

Reply:
I think it's better to show the warning once the command is finished, so that it appears after the list of modified and created files

Also we don't need to show it if the command failed